store/storetest/mocks/UserStore.go

```diff
@@ -848,3 +848,35 @@ func (_m *UserStore) VerifyEmail(userId string) store.StoreChannel {
 
 	return r0
 }
+
+// GetUserInteractions provides a mock function with given fields: uid, teamId, learningGroupPrefixes
+func (_m *UserStore) GetUserInteractions(uid string, teamId string, learningGroupPrefixes []*model.LearningGroupPrefix) store.StoreChannel {
+	ret := _m.Called()
+
+	var r0 store.StoreChannel
```

Comment:
`r0` is only used as a placeholder for a return value. Why not omit it and have:

`if ...`
`    return rf()`
`if ret.Get(0) ...`
`    return ret.Get()...`
`return nil`

That would save some brackets and a local variable.